documentation/content/101 Quick Start/getting-new-releases.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: Getting Notified About New Releases
+Tags: pelican-theme, new-releases
+Category: 101 — Quick Start
+Date: 2019-12-05
+Slug: getting-notified-about-new-releases
+Comment_id: qryv67o-getting-notified-about-new-releases
+authors: Jack De Winter
+---
+
+Unlike a number of themes that exist for Pelican, the Elegant theme is currently under
+active development. The immediate benefit of this for you is that Elegant supports
+recent technologies/services such as [Utterances](https://utteranc.es/) for comments and
+the [Applause button](https://applause-button.com/) for quick and simple user feedback.
+The other main benefits are that any requests, be they bug reports or feature requests,
+are logged in the project's Issues database.
+
+These issues are actively going through triage, being worked on by the Elegant community,
+and being addressed in ongoing releases of Elegant. In addition,
+[the roadmap](https://github.com/Pelican-Elegant/elegant/milestones) let's you see what
+the Elegant team is working on, allowing you to decide if you want to contribute and
+help bring some feature of the theme to life!
+
+To give you the control to decide whether to update your website and benefit from the
+latest version of Elegant, we offer two automated ways to find out about new releases.
+
+## Subscribe to the email Newsletter
+
+Each of the article pages on this Elegant documentation website contain a section at
```

Comment:
Add a line please, something like

> See the "Get updates" section in the sidebar? Enter your email address and click Subscribe button...

Reply:
am I missing something?  This is what I see:

![image](https://user-images.githubusercontent.com/25124582/71328924-b87b2900-24d3-11ea-8d7f-d94f1d68f47d.png)

Reply:
Changed wording to be a bit closer to what you requested.  Waiting for response to above image before progressing forward.